parent/pom.xml

```diff
@@ -66,6 +66,35 @@
         <cxf.osgi.symbolic.name>${project.groupId}.${project.artifactId}</cxf.osgi.symbolic.name>
         <cxf.osgi.dynamic.import />
 
+        <!-- OSGi related properties -->
+        <cxf.fragment.host />
+        <cxf.bundle.activation.policy>lazy</cxf.bundle.activation.policy>
+        <cxf.bundle.activator />
+        <cxf.osgi.import />
+        <cxf.osgi.symbolic.name>${project.groupId}.${project.artifactId}</cxf.osgi.symbolic.name>
+        <cxf.osgi.dynamic.import />
+        <cxf.osgi.asm.version>[9.5,10)</cxf.osgi.asm.version>
+        <cxf.osgi.spring.version>[6,)</cxf.osgi.spring.version>
```

Comment:
Karaf doesn't seem to accept this version range, indeed I get an error of type:
```
Unable to build resource for mvn:org.apache.cxf/cxf-core/4.0.3: invalid version range: [6.0.0,0.0.0)
```
Using simply `6` if it is for 6+ works fine

Reply:
Thanks @essobedo, should be fixed by https://github.com/apache/cxf/commit/bf601f569546086faa081b2d6b64c62032ba0d49